js/blockchaincom.js

```diff
@@ -839,12 +839,15 @@ module.exports = class blockchaincom extends Exchange {
     }
 
     async withdraw (code, amount, address, tag = undefined, params = {}) {
+        if (address !== null) {
+            throw new ExchangeError ("This method doesn't support address, you need to send it in null and add beneficiary attribute in params.");
```

Comment:
This method doesn't support the address. Provide beneficiary attribute in params instead.